src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -19,8 +19,10 @@ const ConfirmationDialog: React.FC<Props> = ({ open, title, body, onConfirm, onC
   const { t } = useTranslation('common');
 
   return (
-    <Dialog open={open} onClose={onClose}>
-      <h2 className={styles.title}>{title}</h2>
+    <Dialog open={open} onClose={onClose} role="dialog" aria-labbeledby="confirmation-heading">
+      <h2 className={styles.title} id="confirmation-heading">
```

Comment:
Theoretically, multiple dialogs can stack. We can use the `useOpaqueId` hook for generating a unique id per rendered dialog.

```tsx
  const headingId = useOpaqueId('confirmation-heading');
```

```suggestion
      <h2 className={styles.title} id={headingId}>
```